config/forcedPhotCcd.py

```diff
@@ -2,8 +2,10 @@
 
 root.measurement.load(os.path.join(os.environ['OBS_SUBARU_DIR'], 'config', 'apertures.py'))
 root.measurement.load(os.path.join(os.environ['OBS_SUBARU_DIR'], 'config', 'kron.py'))
-root.measurement.load(os.path.join(os.environ['OBS_SUBARU_DIR'], 'config', 'cmodel.py'))
+# Turn off cmodel until latest fixes (large blends, footprint merging, etc.) are in
+# root.measurement.load(os.path.join(os.environ['OBS_SUBARU_DIR'], 'config', 'cmodel.py'))
 
 # Disable Gaussian mags (which aren't really forced)
 root.measurement.algorithms.names -= ["base_GaussianFlux"]
+root.measurement.algorithms.names |= ["base_SdssShape"]
```

Comment:
I don't think we should be running `SdssShape` in forced mode, either here or in {{forcedPhotCoadd.py}}, below; it's not actually a forced measurement, and it doesn't look like we're running it there on the HSC side.  Did you find that you had to run it for some reason, or is my assertion that we're not running it on HSC incorrect?

And actually, It think we can remove the line above (and again in `forcedPhotCoadd.py`), since we've fixed `GaussianFlux` on the LSST side such that it actually *does* do forced photometry (it's not PSF-corrected, but neither are the aperture magnitudes).